### DISCUSSION TITLE
Possible to have multiple storage engines?

### DISCUSSION THREAD

**Discussion by spuder** [lra/mackup#2010] [Q&A]
I have a unique situation where I want to backup most of my files to iCloud, but a couple of them I want backed up to a folder

```bash
[storage]
engine = icloud

[applications_to_sync]
aldente

[storage]
engine = file_system
directory = /tmp
```

This appears to not be nativley possible

```bash
configparser.DuplicateSectionError: While reading from '/Users/spuder/.mackup.cfg' [line 13]: section 'storage' already exists
```

Is there another way to support multiple storage locations? 

I've considered using multiple `mackup.cfg` files, but that appears to not be valid

```bash
mackup backup --file ~/.mackup-main
mackup backup --file ~/.mackup-secondary
```

## Reasoning

I want to keep my work gpgkey / ssh keys separate from my personal keys.

**Comment by germanfrelo** [comment]
I was going to ask the same today. My use case is that I would like to have most of my apps settings in iCloud, but the config files for development (git, zsh, etc.) in my custom directory (`~/.dotfiles/Mackup`).
